ci-operator/infra/openshift/ci-chat-bot/deploy.yaml

```diff
@@ -87,3 +87,5 @@ items:
           - /usr/bin/ci-chat-bot
           - --prow-config=/etc/prow/config.yaml
           - --job-config=/etc/jobs
+          - --github-endpoint=http://ghproxy
```

Comment:
Err what, why?

Reply:
Because the chat bot has to look at details of PRs.

Reply:
https://github.com/openshift/ci-chat-bot/blob/master/manager.go#L607